docs/releases/1.9.2.txt

```diff
@@ -71,3 +71,7 @@ Bugfixes
 
 * Fixed a crash when using a reverse ``OneToOneField`` in
   ``ModelAdmin.readonly_fields`` (:ticket:`26060`).
+
+* Fixed a crash when calling the ``migrate`` command in a test case with the
```

Comment:
test case -> TransactionTestCase?
description seems okay otherwise -- did you run across this in one of your projects? (I just wondered if `available_apps` is used outside Django's test suite).